src/reducers/index.ts

```diff
@@ -169,13 +169,20 @@ const contentFiltersReducer = (contentFilters = List<ContentFilter>(), action: A
 const feedsReducer = (feeds = List<Feed>(defaultFeeds), action: Actions): List<Feed> => {
     switch (action.type) {
         case 'ADD-FEED': {
```

Comment:
Ok, so I understand what's going on here. `ADD-FEED` doesn't just add a feed, it also involves some logic, if the feed is already added then it will update the existing feed to be followed.
In that case `ADD-FOLLOWED-FEED` or something similar would be a better name, because by convention all the existing ADD-* actions are just adding to a list. Also the name of the function should be changed, because this is very confusing.